### DISCUSSION TITLE
How to delete ingested Documents?

### DISCUSSION THREAD

**Discussion by TobiasJu** [imartinez/privateGPT#1605] [Q&A]
So i ingested a bunch of documents and want to remove some of them, how can i achieve this?

A button to remove documents in the UI would be handy (and an api endpoint)!

**Comment by TobiasJu** [comment]
I just saw this pull request 🤩
https://github.com/imartinez/privateGPT/pull/1568

**Comment by FabrizioA** [comment]
make wipe
